docs/features/pixi_global_manifest.md

```diff
@@ -0,0 +1,143 @@
+# Pixi Global Manifest
+
+### Motivation
+
+`pixi global` is currently limited to imperatively managing CLI packages.
+The next iteration of this feature should fulfill the following needs:
+
+- Shareable global environments.
+- Allow managing complex environments with multiple packages as dependencies
+- Flexible exposure of binaries
+
+## Design Considerations
+
+There are a few things we wanted to keep in mind in the design:
+
+1. **User-friendliness**: Pixi is a user focussed tool that goes beyond developers. The feature should have good error reporting and helpful documentation from the start.
+2. **Keep it simple**: The CLI should be all you need to interact with the global environments.
+3. **Unsurprising**: Simple commands should still behave similar to traditional package managers.
+4. **Human Readable**: Any file created by this feature should be human-readable and modifiable.
+
+## Manifest
```

Comment:
Would this manifest be a single file or could there be more?

I would see a situation where you would want to import manifests or keep them in separate files. e.g.:

- `pixi-global-company-tools.toml`
- `pixi-global-from-my-dotfiles.toml`
- `pixi-global.toml`: Default

Reply:
I like the idea of multiple pixi-globals, but what would be the discovery order or priority? 
I was thinking more like .git is storing configs,

so I can have a `pixi-global.toml` in `/projects/oss`, so If I'm running `pixi global ` under this path, this will be picked.

If I have  `pixi-global-work.toml` in `/projects/work`running `pixi global ` under this path `pixi-global-work.toml` will be picked.

Reply:
do we need also a new cli command to list all pixi-global and their paths? ( like pixi global list does for packages )

Reply:
Lets put the manifest in `~/.pixi/global/manifests/pixi-global.toml` to future proof the initial implementation.